docs/javascript/v2/features/error-handling.mdx

```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
+| **1003**       | Websocket disconnected - Happens due to network issues                                                    | Mention user to check their network connection or try again after some time.                                |
+| **2002**       | Invalid Endpoint URL                                                                                      | Check the endpoint provided while calling `join` on `HMSSDK`.                                               |
+| **2003**       | Endpoint is not reachable                                                                                 | Mention user to check their network connection or try again after some time.                                |
+| **2004**       | Token is not in proper JWT format                                                                         | The token passed while calling `join` is not in correct format. Retry getting a new token.                  |
+| **3000**       | Overconstrained error                                                                                     | No Action needed from end user, we will be internally retrying.                                             |
+| **3001**       | User denied permission to access capture device at browser level                                          | Ask user to check permission granted to audio/video capture devices in browser(in address bar).             |
+| **3002**       | Capture Device is not Available                                                                           | Ask user to check if the audio/video capture device is connected or not.                                    |
+| **3003**       | Capture device is in use by some other application                                                        | Show notification to user mentioning that the capturing device is used by some other application currently. |
+| **3005**       | There is no media to return. Please select either video or audio or both.                                 | TODO                                                                                                        |
+| **3008**       | Browser has throw an autoplay exception                                                                   | Show notification to user mentioning that the browser blocked autoplay                                      |
+| **3009**       | Device unable to satisfy requested media constraints                                                      | Reduce media constraints for the role in dashboard or upgrade device hardware.                              |
+| **3010**       | No audio detected from track, indicates problem with device hardware                                      | TODO                                                                                                        |
```

Comment:
Notify user to try using a headphone for microphone. @eswarclynn this is for mic only right?

Reply:
Yep it's only for Audio/